driver-core/src/test/java/com/datastax/driver/core/CCMBridge.java

```diff
@@ -685,7 +728,7 @@ public Builder withCassandraConfiguration(String key, String value) {
         }
 
         public CCMBridge build() {
-            CCMBridge ccm = new CCMBridge();
+            CCMBridge ccm = new CCMBridge(isDSE);
             ccm.execute(buildCreateCommand());
             ccm.updateConfig(cassandraConfiguration);
```

Comment:
Need to add a `withDSEConfiguration` method and call `updateDSEConfig` as appropriate.